osf/migrations/0137_add_fm_record_to_osfstorage_files.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.13 on 2018-09-26 15:18
+from __future__ import unicode_literals
+
+from django.db import migrations, connection
+
+
+def remove_records_from_files(state, schema):
+    FileMetadatRecord = state.get_model('osf', 'filemetadatarecord')
+    FileMetadatRecord.objects.all().delete()
+
+def add_records_to_files_sql(state, schema):
+    FileMetadataSchema = state.get_model('osf', 'filemetadataschema')
+    datacite_schema_id = FileMetadataSchema.objects.filter(_id='datacite').values_list('id', flat=True)[0]
+    sql = """
+    INSERT INTO osf_filemetadatarecord (created, modified, _id, metadata, id, file_id, schema_id)
+    SELECT  NOW(), NOW(), generate_object_id(), '{}', nextval('osf_filemetadatarecord_id_seq'), OSF_FILE.id, %d
```

Comment:
Do we need to provide the value for the FileMetadataRecord.id? ( nextval('osf_filemetadatarecord_id_seq'))

Reply:
I think this is already happening!

```
    INSERT INTO osf_filemetadatarecord (created, modified, _id, metadata, id, file_id, schema_id)
    SELECT  NOW(), NOW(), generate_object_id(), '{}', nextval('osf_filemetadatarecord_id_seq'), OSF_FILE.id, %d
```

But maybe I am missing something?

Reply:
geez, I wasn't clear at all, I meant, can we take it away? I thought ids were inserted automatically.

Reply:
Oh oh oh I see YES! Tried removing that and it worked, thank you!